Model/Ship.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vsite.Oom.Battleship.Model
+{
+    public class Ship
+    {
+        public Ship(IEnumerable<Square> squares)
+        {
+            squares = squares;
```

Comment:
Mora biti:
`this.squares = squares;`
Čitajte upozorenja (warninge) kompajlera!